src/menu.rs

```diff
@@ -10,6 +10,7 @@ pub struct Menu {
     text: CloneCell<String>,
     fg: Color,
     entries: Vec<Box<Entry>>,
+    activated: CopyCell<bool>,
```

Comment:
But I stand here and ask **why**? I mean, why not just use mutability directly. cc @jackpot51.